tests/server/cipher.tests.js

```diff
@@ -0,0 +1,37 @@
+import expect from 'expect';
+
+import Cipher from '../../server/lib/cipher';
+
+const password = 'testpwd';
+const cipher = new Cipher(password);
+let encoded;
+
+describe('cipher', () => {
```

Comment:
seems like we're missing unit tests for `decryptString`, `prepareData`, and `processData`

Reply:
we're doing those as part of `unifyData` tests